packages/common/src/ui/components/NavLink.tsx

```diff
@@ -22,21 +22,24 @@ const useStyles = makeStyles(theme => ({
 }));
 
 interface ListItemLinkProps {
-  to: string;
-  icon: JSX.Element;
+  end?: boolean; // denotes lowest level menu item, using terminology from useMatch
+  icon?: JSX.Element;
   text?: string;
+  to: string;
 }
 
 export const AppNavLink: FC<ListItemLinkProps> = props => {
   const classes = useStyles();
-  const selected = useMatch({ path: props.to });
+  const { end, icon = <span style={{ width: 20 }} />, text, to } = props;
```

Comment:
the spacer might need replacing at some point? In which case I thought to move it up here